.gitlab-ci.yml

```diff
@@ -165,23 +165,23 @@ variables:
   # To use images from datadog-agent-buildimages dev branches, set the corresponding
   # SUFFIX variable to _test_only
   DATADOG_AGENT_BUILDIMAGES_SUFFIX: ""
-  DATADOG_AGENT_BUILDIMAGES: v26894465-ddbd51cf
+  DATADOG_AGENT_BUILDIMAGES: v27246432-de27d514
   DATADOG_AGENT_WINBUILDIMAGES_SUFFIX: ""
-  DATADOG_AGENT_WINBUILDIMAGES: v26894465-ddbd51cf
+  DATADOG_AGENT_WINBUILDIMAGES: v27246432-de27d514
   DATADOG_AGENT_ARMBUILDIMAGES_SUFFIX: ""
-  DATADOG_AGENT_ARMBUILDIMAGES: v26894465-ddbd51cf
+  DATADOG_AGENT_ARMBUILDIMAGES: v27246432-de27d514
   DATADOG_AGENT_SYSPROBE_BUILDIMAGES_SUFFIX: ""
-  DATADOG_AGENT_SYSPROBE_BUILDIMAGES: v26563980-7cd1bd3a
+  DATADOG_AGENT_SYSPROBE_BUILDIMAGES: v27246432-de27d514
   DATADOG_AGENT_KERNEL_MATRIX_TESTING_BUILDIMAGES_SUFFIX: ""
-  DATADOG_AGENT_KERNEL_MATRIX_TESTING_BUILDIMAGES: v26473479-2dfb49af
+  DATADOG_AGENT_KERNEL_MATRIX_TESTING_BUILDIMAGES: v27246432-de27d514
   DATADOG_AGENT_NIKOS_BUILDIMAGES_SUFFIX: ""
-  DATADOG_AGENT_NIKOS_BUILDIMAGES: v26473479-2dfb49af
+  DATADOG_AGENT_NIKOS_BUILDIMAGES: v27246432-de27d514
   DATADOG_AGENT_BTF_GEN_BUILDIMAGES_SUFFIX: ""
-  DATADOG_AGENT_BTF_GEN_BUILDIMAGES: v26473479-2dfb49af
+  DATADOG_AGENT_BTF_GEN_BUILDIMAGES: v27246432-de27d514
   # To use images from test-infra-definitions dev branches, set the SUFFIX variable to -dev
   # and check the job creating the image to make sure you have the right SHA prefix
   TEST_INFRA_DEFINITIONS_BUILDIMAGES_SUFFIX: ""
-  TEST_INFRA_DEFINITIONS_BUILDIMAGES: db7cdb8d8c3a
+  TEST_INFRA_DEFINITIONS_BUILDIMAGES: 7a7f90428438
```

Comment:
<!-- Warning highlight specific problems with the subject under review. These problems can be user-facing or behind the scenes and they are blockers. It is strongly recommended to pair this comment with a suggestion. If you are not sure if a problem exists or not, consider leaving a question. -->
**🔨 warning**
This requires a bump in `test/new-e2e/go.mod` side too, opening a PR to handle that